sklearn/preprocessing/label.py

```diff
@@ -37,6 +37,125 @@
 ]
 
 
+def _encode_numpy(values, uniques=None, encode=False):
```

Comment:
Add a note that this should be accessed through `_encode`, where parameters are described.